tasks-by-users/AbbasShaikh0304/05-Css Grid Layout/style.css

```diff
@@ -0,0 +1,40 @@
+body {
```

Comment:
```suggestion
* {
margin: 0;
padding:0;
box-sizing: border-box;
}
```

Reply:
Taken your suggestion in account. Please check commits @Tanvir04khan